scripts/restore-workspace-deps.js

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env node
+/**
+ * Restore @navikt/ft-* dependencies back to "workspace:^"
+ * for all packages in ./packages/*.
+ *
+ * ✅ Updates only dependencies and devDependencies
+ * 🚫 Skips peerDependencies
+ * 🗂️  Should be placed in ./scripts
+ *
+ * Usage:
+ *   node scripts/restore-workspace-deps.js
+ */
+
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
```

Comment:
sonar klager på disse:
```suggestion
import fs from 'node:fs';
import path from 'node:path';
import { fileURLToPath } from 'node:url';
```